### ISSUE TITLE
Disable Jenkins test report comments on GitHub

### ISSUE THREAD

**Issue by msteiger** [MovingBlocks/Terasology#2052]
Since GitHub has now proper support for automated testing reports, we can disable the comment function for the Jenkins PR builder plugin.

**Comment by Cervator** [comment]
It does? Link?

I thought we were still missing some summary info in the comments from Gooey, had hoped there was a way to include the other code metrics beyond unit tests. And still need to hook up the code coverage site fully, arrr. Time.

**Comment by msteiger** [comment]
Imho the automated report generate too much spam and all the relevant info is already in the GitHub summary:

![image](https://cloud.githubusercontent.com/assets/1820007/11809853/f64c6760-a329-11e5-8f8f-3488ea41e9ae.png)

Maybe we can add more info to those GitHub checks later.

**Comment by Cervator** [comment]
Yeah I would be happy to just fit everything into the nice summary setup. Last I checked it just didn't hold a whole lot. Not that the Gooey comment includes a lot of stuff either just yet :-)

I think we can enhance either option, but yeah the nice summary is better. I'm sure we can customize it in all kinds of ways. Just need to find the time, arr :-)

**Comment by msteiger** [comment]
Hmm .. can't find the text message in the Jenkins PR builder config.

**Comment by Cervator** [comment]
Looks like there's a ton of updates to the plugin. Haven't really done a Jenkins overhaul lately.

And I've got 2.5 weeks straight to do that now, woo! Will look at a Jenkins + plugin upgrade early on and pay some extra attention to the PR builder :-)

**Comment by msteiger** [comment]
We should really do this. It spams way too much for my taste. What do you think?

**Comment by rzats** [comment]
From my experience these are useful as automatic notifications for new commits being pushed to a PR branch - is there an alternative notification method that can be set up within GitHub?
The actual report contents (check status & Jenkins link) are indeed obsolete.

**Comment by Cervator** [comment]
I think it is a very subjective topic. Yeah the notifications don't really add any value with the lack of content but on the other hand we might not notice PRs having updated without the comment.

In the perfect world the singular status check block would contain more details (unit test counts, code coverage, etc) which may only be doable as comments (unsure) and rather than comments triggering email spam we'd deliver notifications into a "spammy" notifications channel in Slack. Then you could choose whether or not to listen to that channel.

Checking the current setup we don't get notifications in Slack on new commits in PRs, probably because the commit hook that triggers would trigger in the source repo, not in the target repo. Theoretically *something* does trigger, which is what the PR builder listens to. If we took out the comment we'd still get the Jenkins note that a PR build ran, but it offers only generic info.

There's bound to be *some* way to add new-commit-on-PR notifications to Slack somehow, but it might take a bit of looking.
